e2e/integration/pagination.spec.js

```diff
@@ -63,17 +63,17 @@ describe('Pagination', () => {
     });
 
     it('Set page using URL',() => {
-      cy.visit('/?p=4');
+      cy.visit('/?p=2');
       cy.algoliaQueryRequest('algoliaRequest')
-        .get(Elements.paginatorLink(4)).should('have.class','font-bold');
-      cy.get('@firstBookTitle').contains('Anatomy 337 eReader');
+        .get(Elements.paginatorLink(2)).should('have.class','font-bold');
+      cy.get('@firstBookTitle').contains('Physics 103 and 104 Teaching Guide');
```

Comment:
Had to change the page here. For some reason it was giving timeout to load page 4 - I think it was related to image size